Description: add framapic as alternative upload server
 .
 handylinux-thunar-actions (0.3-1) unstable; urgency=medium
 .
   * fix and publish
Author: arnault perret <arpinux@member.fsf.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- handylinux-thunar-actions-0.3.orig/scripts/pixup
+++ handylinux-thunar-actions-0.3/scripts/pixup
@@ -26,13 +26,26 @@ esac
 for i in $@; do
     TYPE="$(file --mime-type -b $i)"
     EXT="${i##*.}"
-    URL=$(curl -L --progress-bar -F "MAX_FILE_SIZE=15360000" -F "img=@$i;type=$TYPE;filename=$(basename \"$i\")" "http://pix.toile-libre.org/?action=upload" |grep -E -o "http://pix.toile-libre.org/\?img=[^<]*.$EXT" |head -n1)
+
+    # premier essai sur toile-libre
+    URL=$(curl -L --progress-bar -F "MAX_FILE_SIZE=15360000" \
+        -F "img=@$i;type=$TYPE;filename=$(basename \"$i\")" \
+        "http://pix.toile-libre.org/?action=upload" |grep -E -o "http://pix.toile-libre.org/\?img=[^<]*.$EXT" |head -n1)
+
+    if [ $? -ne 0 ]; then
+        #echec avec toile-libre, on essaie framapic
+        URL=$(curl -F "file=@$i" -F "format=$TYPE"\
+            -F "delete-day=0" -F "first-view=0"\
+            "https://framapic.org" |grep "View link" | grep -Eo "https://framapic.org/(.*?)" | cut -d'"' -f1)
+    fi
+
     if [ $? -eq 0 ]; then
         notify-send "$IMGSENT : $i" &
         x-www-browser $URL &
     else
         notify-send "$ERROR" &
     fi
+
 done
 
 IFS=$v_
